src/configuration/services/redis.md

```diff
@@ -4,17 +4,43 @@ Redis is a high-performance in-memory object cache, well-suited for application
 
 See the [Redis documentation](https://redis.io/documentation) for more information.
 
+Platform.sh supports two different Redis configurations: One persistent (useful for key-value application data) and one in-memory-only (useful for application caching).  Aside from that distinction they are identical.
```

Comment:
Currently there are three adjective terms for the same thing: "in-memory-only", "cache only", and "Ephemeral" - just use one term, and define it here at the top